src/content/en/tools/chrome-user-experience-report/index.md

```diff
@@ -0,0 +1,134 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+
+{# wf_updated_on: 2017-10-23 #}
+{# wf_published_on: 2017-10-23 #}
+
+# Chrome User Experience Report {: .page-title }
+
+<img src="images/dataset.png" class="attempt-right" alt="">
```

Comment:
missing alt?

Reply:
I left it blank intentionally. It's just an icon and doesn't add anything to the content of the page, so if it doesn't load or if it's not read by screen readers, that's ok.

![image](https://user-images.githubusercontent.com/1120896/32243630-398b2e6e-be4c-11e7-9683-b3d739b20cd8.png)

BTW is there anything we can do to prevent it from overlapping with the note below?